.github/workflows-source/ci-main.yml

```diff
@@ -117,6 +117,15 @@ jobs:
           if-no-files-found: error
           path: |
             bazel-targets
+      - name: Upload bazel-bep
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.job }}-bep
+          retention-days: 90
+          if-no-files-found: ignore
```

Comment:
in what case do we expect this file not to exist?

Reply:
One example would be change in README on in yaml files. See last workflow run.

Reply:
thanks, can you leave a note?

Reply:
Sure. Does it have to go to every job that has this step?

Reply:
no no, first one should do